README.md

```diff
@@ -1,2 +1,2 @@
-# 06-tcp-server
```

Comment:
It looks like you updated your README to have the title of 09-persistence; however, this is lab 6! You also want to fill out your README with info on your TCP server code